test/functional/Java9andUp/src/org/openj9/test/java/lang/Test_Fma.java

```diff
@@ -0,0 +1,549 @@
+/*******************************************************************************
+ * Copyright (c) 2001, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package org.openj9.test.java.lang;
+
+import org.testng.annotations.Test;
+import org.testng.AssertJUnit;
+
+@Test(groups={ "level.sanity" }, invocationCount=2)
+public class Test_Fma 
+    {
+    @Test
+	public void test_fma_double() 
```

Comment:
Similar to the naming of the file, I think `test_Math_fma_double` to make the package explicit is a bit clearer.  Same comment applies to the Float and StrictMath test cases as well.

Reply:
Fixed in 1907302